distro.py

```diff
@@ -672,8 +673,7 @@ def __repr__(self):
             "_os_release_info={self._os_release_info!r}, " \
             "_lsb_release_info={self._lsb_release_info!r}, " \
             "_distro_release_info={self._distro_release_info!r}, " \
-            "_uname_info={self._uname_info!r})".format(
-                self=self)
+            "_uname_info={self._uname_info!r})".format(self=self)
```

Comment:
With my picky hat on: If this change is needed for a green CI, I vote to move to a dedicated commit. If it's not needed for a green CI, I vote for reverting. Thanks!

Reply:
Reverted.